lib/screens/picture_details_screen/picture_details_screen.dart

```diff
@@ -46,7 +46,7 @@ class _PictureDetailsScreenState extends State<PictureDetailsScreen> {
 
   Future<void> _initScreenInfo() async {
     await _getAnimalName(widget.imagePath);
-    if (animalName != "no name") {
+    if (animalName != "") {
```

Comment:
Won't this make it so if the animal isn't recognized then the details page will just be blank? If so can we add an else statement here so that if `animalName` is empty it sets it to something like "Not Recognized"?